superset/views/base.py

```diff
@@ -420,7 +420,7 @@ def refresh_csrf_token(ex: CSRFError) -> FlaskResponse:
 
 @superset_app.errorhandler(HTTPException)
 def show_http_exception(ex: HTTPException) -> FlaskResponse:
-    logger.warning(ex)
+    logger.warning("HTTP Exception", exc_info=True)
```

Comment:
Nit, for consistency I'd leave this as `HTTPException`, similar to how you did `SupersetErrorsException` — might be easier to grep.

```suggestion
    logger.warning("HTTPException", exc_info=True)
```